ee/src/main/java/org/jboss/as/ee/metadata/EJBClientDescriptorMetaData.java

```diff
@@ -46,6 +48,7 @@
     private int defaultCompression;
 
     private Map<String, RemotingReceiverConfiguration> remotingReceiverConfigurations = new HashMap<String, RemotingReceiverConfiguration>();
+    private List<HttpConnectionConfiguration> httpConnectionConfigurations = new ArrayList<>();
```

Comment:
This field should be final.  The ones above and below as well.

Reply:
Also, there is no code that modifies this list, so something is missing somewhere.

Reply:
This was a bug. I have added the code that allows to configure http connections from jboss-ejb-client.xml file and added a test for that EjbOverHttpDescriptorTestCase